File: app/src/main/java/idev/gin/nias/data/model/POST_SKORING.java
package idev.gin.nias.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class POST_SKORING {

    @SerializedName("kontak_tb")
    @Expose
    private String kontakTb;
    @SerializedName("uji_tuberkulin")
    @Expose
    private String ujiTuberkulin;
    @SerializedName("berat_badan")
    @Expose
    private String beratBadan;
    @SerializedName("demam")
    @Expose
    private String demam;
    @SerializedName("batuk_kronik")
    @Expose
    private String batukKronik;
    @SerializedName("pembesaran_kelenjar")
    @Expose
    private String pembesaranKelenjar;
    @SerializedName("pembengkakan_tulang")
    @Expose
    private String pembengkakanTulang;
    @SerializedName("foto_toraks")
    @Expose
    private String fotoToraks;
    @SerializedName("fk_register")
    @Expose
    private String fkRegister;

    public String getKontakTb() {
        return kontakTb;
    }

    public void setKontakTb(String kontakTb) {
        this.kontakTb = kontakTb;
    }

    public String getUjiTuberkulin() {
        return ujiTuberkulin;
    }

    public void setUjiTuberkulin(String ujiTuberkulin) {
        this.ujiTuberkulin = ujiTuberkulin;
    }

    public String getBeratBadan() {
        return beratBadan;
    }

    public void setBeratBadan(String beratBadan) {
        this.beratBadan = beratBadan;
    }

    public String getDemam() {
        return demam;
    }

    public void setDemam(String demam) {
        this.demam = demam;
    }

    public String getBatukKronik() {
        return batukKronik;
    }

    public void setBatukKronik(String batukKronik) {
        this.batukKronik = batukKronik;
    }

    public String getPembesaranKelenjar() {
        return pembesaranKelenjar;
    }

    public void setPembesaranKelenjar(String pembesaranKelenjar) {
        this.pembesaranKelenjar = pembesaranKelenjar;
    }

    public String getPembengkakanTulang() {
        return pembengkakanTulang;
    }

    public void setPembengkakanTulang(String pembengkakanTulang) {
        this.pembengkakanTulang = pembengkakanTulang;
    }

    public String getFotoToraks() {
        return fotoToraks;
    }

    public void setFotoToraks(String fotoToraks) {
        this.fotoToraks = fotoToraks;
    }

    public String getFkRegister() {
        return fkRegister;
    }

    public void setFkRegister(String fkRegister) {
        this.fkRegister = fkRegister;
    }

    @Override
    public String toString() {
        return "POST_SKORING{" +
                "kontakTb='" + kontakTb +
                ", ujiTuberkulin='" + ujiTuberkulin +
                ", beratBadan='" + beratBadan +
                ", demam='" + demam +
                ", batukKronik='" + batukKronik +
                ", pembesaranKelenjar='" + pembesaranKelenjar +
                ", pembengkakanTulang='" + pembengkakanTulang +
                ", fotoToraks='" + fotoToraks +
                ", fkRegister='" + fkRegister +
                '}';
    }
}

File: app/src/main/java/idev/gin/nias/dao/POST_SKORING.kt
package idev.gin.nias.dao

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class POST_SKORING {

    @SerializedName("kontak_tb")
    @Expose
    var kontakTb: String? = null
    @SerializedName("uji_tuberkulin")
    @Expose
    var ujiTuberkulin: String? = null
    @SerializedName("berat_badan")
    @Expose
    var beratBadan: String? = null
    @SerializedName("demam")
    @Expose
    var demam: String? = null
    @SerializedName("batuk_kronik")
    @Expose
    var batukKronik: String? = null
    @SerializedName("pembesaran_kelenjar")
    @Expose
    var pembesaranKelenjar: String? = null
    @SerializedName("pembengkakan_tulang")
    @Expose
    var pembengkakanTulang: String? = null
    @SerializedName("foto_toraks")
    @Expose
    var fotoToraks: String? = null
    @SerializedName("fk_register")
    @Expose
    var fkRegister: String? = null

    override fun toString(): String {
        return "POST_SKORING{" +
                "kontakTb='" + kontakTb +
                ", ujiTuberkulin='" + ujiTuberkulin +
                ", beratBadan='" + beratBadan +
                ", demam='" + demam +
                ", batukKronik='" + batukKronik +
                ", pembesaranKelenjar='" + pembesaranKelenjar +
                ", pembengkakanTulang='" + pembengkakanTulang +
                ", fotoToraks='" + fotoToraks +
                ", fkRegister='" + fkRegister +
                '}'.toString()
    }
}
